### PR TITLE
Fixes cpr'ing undefibbable bodies

### PR DESCRIPTION
**Pull request by QualityVan**
## About The Pull Request
The undefibbable check was on the wrong person, so you couldn't give cpr if you yourself were undefibbable.
Also increases damage from suicide to ensure that you're below health thresholds.

## Why It's Good For The Game
Bugfix

## Changelog
:cl:
fix: Trying to give cpr to someone who's gone cold will fail properly now
fix: Suiciding gives more oxy damage, just so the body meets damage thresholds for cpr checks
/:cl:

### CODE REVIEW

**Comment by BraveMole** [comment]
very funny

**Review by TiviPlus — APPROVED** [review]

File: code/modules/mob/living/carbon/human/human_attackhand.dm [hunk 1]
```diff
@@ -31,12 +31,12 @@
...
-			if((head && (head.flags_inventory & COVERMOUTH)) || (wear_mask && (wear_mask.flags_inventory & COVERMOUTH)))
-				to_chat(H, span_boldnotice("Remove his mask!"))
+			if(HAS_TRAIT(src, TRAIT_UNDEFIBBABLE ))
+				to_chat(H, span_boldnotice("Can't help this one. Body has gone cold."))
 				return FALSE
 
-			if(HAS_TRAIT(H, TRAIT_UNDEFIBBABLE ))
-				to_chat(H, span_boldnotice("Can't help this one. Body has gone cold."))
+			if((head && (head.flags_inventory & COVERMOUTH)) || (wear_mask && (wear_mask.flags_inventory & COVERMOUTH)))
+				to_chat(H, span_boldnotice("Remove his mask!"))
```
Comment (XSlayer300) [inline code/modules/mob/living/carbon/human/human_attackhand.dm:39]:
```suggestion
				to_chat(H, span_boldnotice("Remove their mask!"))
```

File: code/modules/projectiles/gun_system.dm [hunk 2]
```diff
@@ -844,7 +844,7 @@ and you're good to go.
 			user.apply_damage(200, STAMINA)
 		else
 			user.apply_damage(projectile_to_fire.damage * 2.5, projectile_to_fire.ammo.damage_type, "head", 0, TRUE)
-			user.apply_damage(100, OXY)
+			user.apply_damage(200, OXY)
```
Comment (SplinterGP) [inline code/modules/projectiles/gun_system.dm:847]: undocumented change, put on CL

**Review by SplinterGP — CHANGES_REQUESTED** [review]

File: code/modules/projectiles/gun_system.dm (hunk 2 above)
Comment (BraveMole) [reply code/modules/projectiles/gun_system.dm:847]: > undocumented change, put on CL

image
 ??

Comment (QualityVan) [reply code/modules/projectiles/gun_system.dm:847]: Just edited that in a minute ago.

Comment (BraveMole) [reply code/modules/projectiles/gun_system.dm:847]: ah ok haha

File: code/modules/mob/living/carbon/human/human_attackhand.dm (hunk 1 above)
Comment (TiviPlus) [reply code/modules/mob/living/carbon/human/human_attackhand.dm:39]:
```suggestion
				to_chat(H, span_boldnotice("Remove [p_their()] mask!"))
```

**Review by SplinterGP — APPROVED** [review]

**Pull request merged**